jetson/db/db_connection.py

```diff
@@ -0,0 +1,127 @@
+import mysql.connector
+from .config import get_config
+from contextlib import contextmanager, closing
+
+
+class Table:

```

Comment:
this is fine for now. However i'd like to open an issue to further improve this in the future as we should be able to auto generate some basic SQL queries like insert, and truncate for a table given its definition in code. I can explain in more detail somewhere else.